src/Compilers/VisualBasic/Portable/Binding/MemberSemanticModel.vb

```diff
@@ -794,6 +794,51 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         End Function
 
         Friend Overrides Function GetOperationWorker(node As VisualBasicSyntaxNode, options As GetOperationOptions, cancellationToken As CancellationToken) As IOperation
+            Dim argumentNode = TryCast(node, ArgumentSyntax)
+            If argumentNode IsNot Nothing Then
+                Return TryGetOperationForArgument(options, argumentNode, cancellationToken)
```

Comment:
Should we be returning Nothing in the event of failure to get the operation, or should we be returning GetBoundNodeSummaryAndCreateOperation?